E2_ESRGAN/lib/utils.py

```diff
@@ -164,19 +164,24 @@ class RDB(tf.keras.layers.Layer):
 
   def __init__(self, out_features=32, bias=True):
     super(RDB, self).__init__()
-    self.conv = lambda x: tf.keras.layers.Conv2D(
+    conv= lambda: tf.keras.layers.Conv2D(
```

Comment:
Please make this into a standard private function called for example
_create_conv2d

Reply:
Done :)